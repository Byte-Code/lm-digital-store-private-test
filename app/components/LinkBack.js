import React, { PropTypes } from 'react';
import { hashHistory } from 'react-router';

const LinkBack = (props) => (
  <a onClick={hashHistory.goBack}>
    {props.children}
  </a>
);

LinkBack.propTypes = {
  children: PropTypes.node.isRequired
};

export default LinkBack;
