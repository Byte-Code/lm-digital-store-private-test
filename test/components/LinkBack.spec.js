import React from 'react';
import { shallow } from 'enzyme';

import LinkBack from '../../app/components/LinkBack';

describe('LinkBack', () => {
  it('renders properly', () => {
    const result = shallow(
      <LinkBack>
        <p>children</p>
      </LinkBack>
    );
    expect(result).toMatchSnapshot();
  });
});
