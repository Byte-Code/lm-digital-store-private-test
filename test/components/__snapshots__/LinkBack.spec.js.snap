// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkBack renders properly 1`] = `
<a
  onClick={[Function]}
>
  <p>
    children
  </p>
</a>
`;
